langchain/document_loaders/sitemap.py

```diff
@@ -39,6 +41,52 @@ def __init__(
 
         self.filter_urls = filter_urls
         self.parsing_function = parsing_function or _default_parsing_function
+        if discover_sitemap:
+            self.web_paths = self._modify_web_path()
+
+    # if web_path is a str
+    @property
+    def _base_url(self) -> str:
+        base_url = (
+            f"{urlparse(self.web_path).scheme}"
+            + "://"
+            + f"{urlparse(self.web_path).netloc}"
+        )
+        return base_url
+
+    def _find_sitemap_in_robotstxt(self) -> List[str]:
+        import requests
+
+        """Find sitemap in robots.txt."""
+        sitemap_urls: List[str] = []
```

Comment:
@Pouyanpi This should be `site_map_urls`

Reply:
Thanks for the suggestion. I used it as "sitemap" and is the more commonly used spelling in the context of website architecture and organization, compared to "site map." As a result, using "sitemap_urls" as the variable name might be more fitting and in line with convention. What do you think? Of course, the project's convention has to be followed.

Reply:
Hey sorry for replying so late, actually I was just pointing out a mistyped variable, if you see in this method below line 65 references `site_map_urls` whereas `sitemap_urls` was declared. I think `sitemap_urls` is a more fitting name for sure.